src/hub.html

```diff
@@ -29,6 +29,7 @@
         >
 
         <a-assets>
+            <img id="tooltip"  src="./assets/hud/tooltip.png" >
```

Comment:
Would be nice to indicate in the filename somehow that this is a 9slice. Android uses `.9.png`, so i'd do that unless you have something better.

Reply:
👍